src/main/java/emissary/roll/RollManager.java

```diff
@@ -95,6 +103,19 @@ public void update(Observable o, Object arg) {
     }
 
     public static RollManager getManager() {
+        if (RM == null) {
```

Comment:
Perhaps we should synchronize both `getManager()` and `getManager(Configurator)` so that it's not possible to unintentionally create two roll managers if multiple threads should attempt to call the methods simultaneously and each passes the null check.